.eslintrc.js

```diff
@@ -1,15 +1,22 @@
 module.exports = {
   env: {
-    browser: true
+    browser: true,
```

Comment:
OOC what does this do?

Reply:
Nevermind answered my own questions here: https://eslint.org/docs/user-guide/configuring#specifying-globals